Adding some more information to the readme, #45 (#46)
matomo-org/plugin-Migration: README.md

```
ssh -NL 3307:localhost:3306 targetuser@targethost
```
This command should be run on the source Matomo server. It essentially maps port 3307 of the server to port 3306 of the server you are migrating to and is specified by `targethost`.
The `targethost` should be replaced by a valid IP or domain name referencing the server you are migrating to and must be accessible to the server that you are migrating from.
The `targetuser` should be replaced with a valid SSH user account setup on that server that you are migrating to.

An alternative to using an SSH tunnel is to make a backup of your MySQL database, copy it to the new server, import it into a temporary database, and then migrate using that database name.
Remember to delete the temporary database after completing the migration, and checking that everything works.

> More information about this process can be found in a [FAQ about moving between servers](https://matomo.org/faq/how-to-install/faq_76/).

Reviewer: ```suggestion
More information about this process can be found [here](https://matomo.org/faq/how-to-install/faq_76/).
```

Reviewer 2: I dislike "here" links, and would keep the original version.

If anything, the link text could be the original title: "How can I move Matomo from one server to another, also migrating the data from one mysql server to another?"

Reviewer: Even the FAQ title looks good 👍🏻

Author: Yeah. I'm pretty sure that I read in one of our wiki pages that we should use more descriptive text than just `here` or `FAQ` when linking to things in our documentation. Using the full name of the FAQ will work :+1:
